manager/controlapi/node.go

```diff
@@ -283,7 +283,7 @@ func (s *Server) RemoveNode(ctx context.Context, request *api.RemoveNodeRequest)
 				return grpc.Errorf(codes.FailedPrecondition, "node %s is a cluster manager and is a member of the raft cluster. It must be demoted to worker before removal", request.NodeID)
 			}
 		}
-		if node.Status.State == api.NodeStatus_READY {
+		if !request.Force && node.Status.State == api.NodeStatus_READY {
```

Comment:
@LK4D4 See this portion:

```
if member := s.raft.GetMemberByNodeID(request.NodeID); member != nil {
    return grpc.Errorf(codes.FailedPrecondition, "node %s is a cluster manager and is a member of the raft cluster. It must be demoted to worker before removal", request.NodeID)
}
```

We can't force-remove a Manager as the check on the raft memberlist is upstream to the force remove check. So we can only force-remove Workers and Managers that are not registered in Raft yet (which was handy with the acceptance model). So afaik you still have to demote manually if you want to force remove a node that was a raft member.